FAQ.md

```diff
@@ -0,0 +1,52 @@
+# FAQ - Frequently Asked Questions
+
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+
+**Table of Contents**
+
+- [FAQ - Frequently Asked Questions](#faq---frequently-asked-questions)
+  - [Installation](#installation)
+  - [Trading](#trading)
+  - [Wallet](#wallet)
+  - [Contact us](#contact-us)
```

Comment:
GitHub has a built-in ToC:

![image](https://user-images.githubusercontent.com/5486389/165453356-994a2a9b-3c86-4d92-8cbd-2c7b7a548e34.png)

Reply:
WAAAAAT! I was not aware of that! The one at the top still provides value IMHO, as it is more discoverable (shows the contents straight away)